.github/workflows/nodejs.yml

```diff
@@ -91,3 +91,7 @@ jobs:
         run: yarn test:smoke
         env:
           CI: true
+
+  commitlint:
```

Comment:
> Note: It's necessary that you specify the fetch-depth argument to actions/checkout@v2 step. By default they fetch only latest commit of the branch, but we need more commits since we validate a range of commit messages.

Make this check with steps with first being `actions/checkout` and next being `commitlint`. Refer documentation link  James' comment for more info.